docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -112,28 +111,32 @@ ev_callback(int type, tzsh_quickpanel_event_info_h ev_info, void *data)
 
    tzsh_quickpanel_event_visible_get(ev_info, &state);
    if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN)
-     ;
+     {
+        /* visible state */
+     }
    else if (state == TZSH_QUICKPANEL_STATE_VISIBLE_HIDDEN)
-     ;
+     {
+        /* invisible state */
+     }
    else
-     ;
+     {
+        /* error */
+     }
 }
 
 static void
-init_tzsh(Evas_Object *main_win)
+init(Evas_Object *main_win)
 {
    ...
 
    /* register event callback */
    handler = tzsh_quickpanel_event_handler_add(tzsh_qp, TZSH_QUICKPANEL_EVENT_VISIBLE, ev_callback, NULL);
-
-   ...
 }
 ```
 
 
 ## Destroying TZSH Quickpanel and TZSH Structures
```

Comment:
## Destroy TZSH Quickpanel and TZSH Structures

Reply:
@gl-lee  can we finish our tasks. Kindly accommodate a few latest comments such that I will approve.

Reply:
ok.